core/test/libraries/BalanceSheet.t.sol

```diff
@@ -12,121 +12,155 @@ import {
     PROBE_PERCENT_MAX,
     IV_COLD_START
 } from "src/libraries/constants/Constants.sol";
-import {BalanceSheet, TickMath, square} from "src/libraries/BalanceSheet.sol";
+import {BalanceSheet, Assets, Prices, TickMath, square} from "src/libraries/BalanceSheet.sol";
 
 import {FixedPointMathLib as SoladyMath} from "solady/utils/FixedPointMathLib.sol";
 
+contract LibraryWrapper {
+    function isHealthy(
+        Prices memory prices,
+        Assets memory mem,
+        uint256 liabilities0,
+        uint256 liabilities1
+    ) external pure returns (bool) {
+        return BalanceSheet.isHealthy(prices, mem, liabilities0, liabilities1);
+    }
+}
+
 contract BalanceSheetTest is Test {
     function setUp() public {}
 
+    function test_alwaysHealthyWhenLiabilitiesAre0(
```

Comment:
Since I moved the isHealthy check inside of the `liabilities0 > 0 || liabilities1 > 0` conditional, I wanted to double check that no debt does, in fact, imply you're healthy. This test proves that.